packages/wasm-tests/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "wasm-tests"
+version = "0.1.0"
```

We generally use `0.0.0` for unpublished packages.
```suggestion
version = "0.0.0"
```